app/invocation/invocation_compare_button.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { fromEvent, Subscription } from "rxjs";
+import { OutlinedButton } from "../components/button/button";
+import Menu, { MenuItem } from "../components/menu/menu";
+import Popup from "../components/popup/popup";
+import router from "../router/router";
+import capabilities from "../capabilities/capabilities";
+
+export interface InvocationCompareButtonComponentProps {
+  invocationId: string;
+}
+
+interface State {
+  invocationIdToCompare: string;
+  isDropdownOpen: boolean;
+}
+
+const INVOCATION_ID_TO_COMPARE_LOCALSTORAGE_KEY = "invocation_id_to_compare";
```

Comment:
at some point we should probably collect all of these into a file or service - same with subscribing to changes

Reply:
Agreed